codes/qubits/honeycomb.yml

```diff
@@ -0,0 +1,56 @@
+
+  # transversal_gates: 'Transversal ... gates \cite{doi:ok-paper}. Comment out if doesn''t apply.'
+
+  general_gates:
+   - ‘There are two types of logical operators, ‘’inner’’ and ‘’outer.’’ An inner logical operator is the product of check operators on a homologically nontrivial cycle. They belong to the stabilizer group as a subsystem code. Outer logical operators have an interpretation in terms of magnetic and electric operators of an embedded toric code, and they do not belong to the stabilizer group as a subsystem code.’
+
+  decoders: 
+    - ‘The ISG has a static subgroup for all time steps \(r\geq 3\) – that is, a subgroup which remains a subgroup of the ISG for all future times – given by so-called ‘’plaquette stabilizers.’’ These are stabilizers consisting of products of check operators around homologically trivial paths. The syndrome bits correspond to the eigenvalues of the plaquette stabilizers. Because of the structure of the check operators, only one-third of all plaquettes are measured each round. The syndrome bits must therefore be represented by a lattice in spacetime, to reflect when and where the outcome was obtained.’
+
+  fault_tolerance:
+    - 'One can run a fault-tolerant decoding algorithm by bipartitioning the syndrome lattice into two graphs which are congruent to the Cayley graph of the free abelian group with three generators (up to boundary conditions) are performing a matching algorithm to deduce errors.'
+
+  threshold: 
+    - ‘A Peierls argument shows that there is a positive threshold so that up to this threshold a minimum weight matching algorithm will match errors leaving all loops of residual errors small.’
+
+realizations:
+  - ’The Honeycomb code admits a representation in terms of Majorana fermions, as was used to describe the Kitaev honeycomb model \cite{doi:10.1016/j.aop.2005.10.005}. Such a representation leads to a possible physical realization of the code in terms of tetrons \cite{doi:10.1103/PhysRevB.95.235305}, where each physical qubit is composed of four Majorana modes.’
+
+notes:
+ 
+
+relations:
+  parents:
+    - code_id: dynamical
+      detail: ‘This code belongs to a larger class of codes whose logical qubits are dynamically generated.’
+    - code_id: subsystem
+      detail: ‘This code can be viewed as a subsystem code, albeit one without logical qubits.’
+
+  cousins:
+    - code_id: surface
+      detail: ‘Measurement of each check operator involves two qubits and projects the state of the two qubits to a two-dimensional subspace, which we regard as an effective qubit. These effective qubits form a toric code on a hexagonal superlattice. Electric and magnetic operators on the embedded toric code correspond to outer logical operators of the honeycomb code. In fact, outer logical operators transition back and forth from magnetic to electric toric code operators under the measurement dynamics.’
+
```

Comment:
@mgullans please add any other cousins or parents. Parent here is right now dynamic_gen, but should cousin be quantum_ldpc?

Reply:
Yes, cousin can be quantum LDPC.  Another cousin could be subsystem code since you measure noncommuting operators (except it has no logicals as a subsystem code).

Reply:
@fechisin please add quantum_LDPC and subsystem as cousins. If you're unsure of descriptions, please tag us here.